.dotnet/tests/Images/ImageGenerationMockTests.cs

```diff
@@ -0,0 +1,586 @@
+﻿using System;
+using System.ClientModel;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenAI.Images;
+using OpenAI.Tests.Utility;
+
+namespace OpenAI.Tests.Images;
+
+[TestFixture(true)]
+[TestFixture(false)]
+[Parallelizable(ParallelScope.All)]
+[Category("Smoke")]
+public partial class ImageGenerationMockTests : SyncAsyncTestBase
```

Comment:
`GenerateImageEdit` and `GenerateImageVariation` overloads taking a file path do not support passing a `CancellationToken`. Is this intentional?

Reply:
It is, although I wonder if we should change it. In any case, since CancellationToken is optional, we can add it later.